src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = sourceString
+    .split(';')
+    .map((styles) => styles.trim())
+    .filter((styles) => styles.length > 0)
+    .map((styles) => styles.split(':'))
+    .reduce(
+      (object, styles) => ({
+        ...object,
+        [styles[0].trim()]: styles[1].trim(),
```

Comment:
There is no check to handle cases where there might be multiple colons within the style string's property values. While not required, handling such cases would make the function more robust.